### ISSUE TITLE
[research] BraTS 2023 获取渠道调研

### ISSUE THREAD

**Issue by ACautomata** [ACautomata/NV-Generate-CTMR#4]
Part of #2

## Question

BraTS 2023 Adult Glioma 数据集如何获取？确认格式假设。

需要回答：
1. 权威渠道：Synapse（注册/审批流程、团队加入要求）与替代渠道（HF 镜像、Kaggle 等）对比；推荐最小摩擦路径。
2. 数据格式确认：4 模态（t1 / t1c / t2 / flair）NIfTI 命名约定、分割 mask 命名、是否 skull-stripped、co-registered、1mm³、体素尺寸（240×240×155?）、病例数（Adult Glioma ~1251?）。
3. 使用条款（CC-BY？研究用途限制）。
4. 下载工具（synapseclient 等）与数据量级。

产出：findings 作为本 issue comment 发布（中文、技术名词保留英文），然后关闭本 issue。

**Comment by ACautomata** [comment]
## BraTS 2023 Adult Glioma（GLI）获取渠道调研 findings

### TL;DR 推荐

**最小摩擦路径 = HuggingFace 镜像 [`MedOtter/brats2023-gli-dataset`](https://huggingface.co/datasets/MedOtter/brats2023-gli-dataset)**：已通过 HF API 逐项验证为**完整训练集**（6255 个 `.nii.gz` = 1251 cases × 5 files，目录结构复刻官方 `ASNR-MICCAI-BraTS2023-GLI-Challenge-TrainingData/`），non-gated、免注册、声明 CC-BY-4.0。

```bash
pip install -U "huggingface_hub[cli]"
hf download MedOtter/brats2023-gli-dataset --repo-type dataset \
  --local-dir data/brats2023-gli
# 旧版 CLI 等价：huggingface-cli download MedOtter/brats2023-gli-dataset --repo-type dataset --local-dir data/brats2023-gli
```

下载后建议抽查（社区镜像，无官方 checksum 可比对）：

```python
import nibabel as nib, numpy as np, glob
f = sorted(glob.glob("data/brats2023-gli/ASNR-MICCAI-BraTS2023-GLI-Challenge-TrainingData/*/"))[0]
img = nib.load(f + "BraTS-GLI-00000-000-t1c.nii.gz")
assert img.shape == (240, 240, 155)
seg = np.asarray(nib.load(f + "BraTS-GLI-00000-000-seg.nii.gz").dataobj)
assert set(np.unique(seg)) <= {0, 1, 2, 3}
```

### 1. 渠道对比

| 渠道 | 状态 | 摩擦 | 备注 |
|---|---|---|---|
| **HF 镜像 `MedOtter/brats2023-gli-dataset`** | 完整（已验证 1251 cases × 5 files） | 最低：免注册、一条命令 | 社区镜像（2025-10 上传）；仅含 TrainingData（含 seg） |
| **Synapse 官方** | ⚠️ 原 2023 challenge 站点（[syn51156910](https://www.synapse.org/Synapse:syn51156910)）**已归档，数据不可再从该站下载** | 高 | 现需经 [synapse.org/brats](https://www.synapse.org/brats)（当前 challenge 集群）的 "Request for Annotations or Access to Previous BraTS Dataset" 表单人工申请；流程：Synapse 注册 → 认证 → 同意 terms of use → 提交申请等审批 → `synapseclient` 下载（`pip install synapseclient` → `synapse login -u ... -p ...` → `synapse get <synID> --downloadLocation ...`） |
| **Kaggle 镜像** | [`ishita21gupta/brats-2023-adult-glioma`](https://www.kaggle.com/datasets/ishita21gupta/brats-2023-adult-glioma)（1251 dirs）、[`shakilrana/brats-2023-adult-glioma`](https://www.kaggle.com/datasets/shakilrana/brats-2023-adult-glioma) | 中：需 Kaggle 账号 + API token | `kaggle datasets download -d ishita21gupta/brats-2023-adult-glioma`；完整性未逐文件验证 |
| **TCIA** | ❌ 无 BraTS-GLI 2023 collection（`cancerimagingarchive.net/collection/brats-gli/` 404） | — | TCIA 仅有 BraTS-PEDs / BraTS-Africa 等子集与旧版 RSNA-ASNR-MICCAI-BraTS-2021 |

### 2. 数据格式确认（全部证实）

- **命名约定**：每 case 一个目录，含 5 个文件
  `BraTS-GLI-XXXXX-XXX-{t1n,t1c,t2w,t2f,seg}.nii.gz`
  注意 2023 官方后缀：`t1n`（非 t1）、`t1c`（非 t1ce）、`t2w`（非 t2）、`t2f`（非 flair）。case 尾缀 `-000/-001` 是 timepoint（longitudinal 复扫，训练集绝大多数为 `-000`）。
- **预处理**（[BraTS 2023 官方 wiki](https://www.synapse.org/Synapse:syn51156910/wiki/622553) / [challenge paper](https://pmc.ncbi.nlm.nih.gov/articles/PMC10441440/) 确认）：DICOM→NIfTI、**co-registered**（SRI24 template）、**resampled 1mm³ isotropic**、**skull-stripped** ✅ 假设成立。
- **体素尺寸**：`240×240×155` ✅（抽样文件 shape 验证 + 文献一致）。
- **病例数**：training **1251**（有 seg）+ validation 219（无公开 ground truth）✅（官方 wiki 原文 "a training set of 1251 cases and a validation set of 219 cases"）。HF 镜像 = training 集。
- **⚠️ Segmentation labels**：BraTS 2023 起改为 `0=background, 1=NCR/NET(NETC), 2=ED(SNFH), 3=ET` —— **ET 从 2021 及以前的 label 4 改为 3**。下游解析 mask 时勿沿用旧 {1,2,4} 约定。

### 3. 使用条款

- 数据本身以 **CC BY 4.0** 发布（HF 镜像卡片声明 `cc-by-4.0`；与 BraTS 官方 2021 起的发布许可一致）：允许商用/修改，仅要求 attribution（引用 [BraTS 2023 challenge 论文](https://arxiv.org/abs/2305.09011)）。无"仅限研究"限制。
- 注意区分：challenge 论文**文章文本**是 CC BY-NC-ND，那是出版许可，不约束数据。

### 4. 下载量级

- Training 集（1251 cases × 4 modalities + seg）：抽样单文件 t1n≈2.3MB / t1c≈2.6MB / t2w≈2.1MB / t2f≈2.3MB / seg≈0.05MB → 约 **12 GB**（compressed NIfTI）。磁盘预留 ~25 GB（解压前后）即可。

### 参考来源

- [BraTS 2023 Challenge — Synapse (syn51156910)](https://www.synapse.org/Synapse:syn51156910)（官方站点，已归档）
- [BraTS 2023 官方 wiki：数据与预处理](https://www.synapse.org/Synapse:syn51156910/wiki/622553)
- [HF 镜像 dataset card](https://huggingface.co/datasets/MedOtter/brats2023-gli-dataset)
- [The BraTS Challenge 2023 paper (PMC10441440)](https://pmc.ncbi.nlm.nih.gov/articles/PMC10441440/) / [arXiv:2305.09011](https://arxiv.org/abs/2305.09011)
- [当前 BraTS challenge 集群（历史数据申请入口）](https://www.synapse.org/brats)

**Comment by ACautomata** [comment]
调研完成，findings 见上方 comment。推荐路径：HF 镜像 MedOtter/brats2023-gli-dataset（已验证完整 1251 cases，免注册，CC-BY-4.0，~12GB）。
